qa/tasks/cephfs/kernel_mount.py

```diff
@@ -257,17 +257,18 @@ def _get_global_id(self):
                 import json
 
                 def get_id_to_dir():
-                    result = {}
+                    meta_dir = "{meta_dir}"
+                    result = dict()
                     for dir in glob.glob("/sys/kernel/debug/ceph/*"):
-                        if os.path.basename(dir) == DEBUGFS_META_DIR:
+                        if os.path.basename(dir) == meta_dir:
                             continue
                         mds_sessions_lines = open(os.path.join(dir, "mds_sessions")).readlines()
                         global_id = mds_sessions_lines[0].split()[1].strip('"')
                         client_id = mds_sessions_lines[1].split()[1].strip('"')
                         result[client_id] = global_id
                     return result
                 print(json.dumps(get_id_to_dir()))
-            """)
+            """.format(meta_dir=DEBUGFS_META_DIR))
```

Comment:
nit: Let's use f-string instead?

IMO, `f"{x1} {x2} {x3}"` is more readable than `"x1 x2 x3".format(x1='ab', x2='cd', x3='ef')`

Reply:
While you are absolutely right - this is dated code and I'm _not_ inclined in changing it in this PR.

Reply:
Okay, cool.